packages/core/helpers/gatherQueryParams.ts

```diff
@@ -1,13 +1,19 @@
 import _ from 'lodash'
 
+const isNumber = (value: string) => {

```

Comment:
This looks good! I think early on in the project we discussed adding helpers for strings, dates, numbers, etc in this location. `packages/core/helpers/cove/number.js` I'll leave that up to you though.

Reply:
This check is something that's really kinda specific to the query parameters and I don't have the confidence that it should be used in all scenarios. I'd make it a helper otherwise.